src/hotspot/cpu/aarch64/aarch64_sve.ad

```diff
@@ -340,7 +339,38 @@ instruct storeV_partial(vReg src, vmemA mem, pRegGov pTmp, iRegINoSp tmp1,
   %}
   ins_pipe(pipe_slow);
 %}
+instruct reinterpret(vReg dst) %{
+  predicate(UseSVE > 0 && n->as_Vector()->length_in_bytes() >= 16 &&
+            n->in(1)->bottom_type()->is_vect()->length_in_bytes() >= 16 &&
```

Comment:
This check in line-344 is redundant as the pre and subsequent check can guarantee this.

Reply:
Sure